pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandler.java

```diff
@@ -301,10 +301,6 @@ Map<String, List<Operation>> computeOperations(SegmentDirectory.Reader segmentRe
           continue;
         }
 
-        // Note that RAW columns cannot be sorted.
```

Comment:
nit:

In L757, can we remove the comment and also name the ForwardIndex tmp file appropriately?